x-pack/plugins/security_solution/public/common/components/search_bar/index.tsx

```diff
@@ -91,16 +92,25 @@ export const SearchBarComponent = memo<SiemSearchBarProps & PropsFromRedux>(
       },
     } = useKibana().services;
 
+    const dispatch = useDispatch();
+    const setTablesActivePageToZero = useCallback(() => {
+      dispatch(usersActions.setUsersTablesActivePageToZero());
+      dispatch(hostsActions.setHostTablesActivePageToZero());
+      dispatch(networkActions.setNetworkTablesActivePageToZero());
+    }, [dispatch]);
+
     useEffect(() => {
       if (fromStr != null && toStr != null) {
         timefilter.setTime({ from: fromStr, to: toStr });
       } else if (start != null && end != null) {
+        setTablesActivePageToZero();
```

Comment:
Do we need `setTablesActivePageToZero` in Line 104 as well?

Reply:
In theory, we should. But I can't find a use case where it is necessary. Do you know a use case where the user can change `fromStr` or `toStr` beside the search bar? For example, line 106 is called when we update the time filter by interacting with a histogram.

@angorayc Do you think I should add `setTablesActivePageToZero` to 104 for consistency? It would also add the side-effect that active pages are cleaned when the user navigates between pages because line 104 is called don when search_bar initializes.

Reply:
Got it, I guess I was confused between start and fromStr, end and toStr. If it is only called on navigating between pages, then no need to add it. Thanks Pablo.